tests/filecheck/mlir-conversion/llvm_test.xdsl

```diff
@@ -1,46 +1,49 @@
-// RUN: xdsl-opt -t mlir | xdsl-opt -t mlir -f mlir | filecheck %s
```

Comment:
This testcase seemed to forget that we are printing the generic IR for MLIR. I refactored it s.t. it now knows that.